components/xPoints/index.tsx

```diff
@@ -40,12 +43,25 @@ function Component() {
     const [xpoints_addr, setAddr] = useState(addr)
 
     useEffect(() => {
+        setLoading(true)
         fetchXpoints()
             .then(() => {
-                fetchMotion().then(() => {
-                    setLoading(false)
-                })
-                    .catch(error => { throw error })
+                fetchMotion()
+                    .then(() => {
+                        setLoading(false)
+                    })
+                    .catch((error) => {
+                        toast.error(String(error), {
```

Comment:
can we not throw the error and catch it in the outer catch?

Reply:
Hi @tralkan , yes let me update it